students/minwook/westagram_back/User/views.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from django.views           import View
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from .models import User
+
+class SignUp(View):
```

Comment:
```suggestion
class SignUpView(View):
```
View 이름에는 `View`를 붙이시는 것을 추천드립니다.

Reply:
모든 View 클래스 이름에 View 붙이는 걸로 변경했습니다.